### DISCUSSION TITLE
Day 2 Updates

### DISCUSSION THREAD

**Discussion by Chandra-Mauli-Sharma** [Safe-Bump/Safe-Bump-App#1] [General]
I have set up the project and started implementing the UI. The Login Screen is done. I will create a Firebase project for this to integrate.

Just for Updates. I am using 

- **Get It** -> as dependency injection framework 
- **Provider** -> for state management

@devprashantt & @AdityaChaudhary18, Any updates from your side?

**Comment by prashant-software-developer** [comment]
> I have set up the project and started implementing the UI. The Login Screen is done. I will create a Firebase project for this to integrate.
> 
> Just for Updates. I am using
> 
> * **Get It** -> as dependency injection framework
> * **Provider** -> for state management
> 
> @devprashantt & @AdityaChaudhary18, Any updates from your side?

I have created some more screens for user :- 

- Dashboard screen
- Articles and videos educational screen
- Suggested food screen
- Calendar screen

Working on health screen, that's all from my side

**Comment by AdityaChaudhary18** [comment]
> I have set up the project and started implementing the UI. The Login Screen is done. I will create a Firebase project for this to integrate.
> 
> Just for Updates. I am using
> 
> * **Get It** -> as dependency injection framework
> * **Provider** -> for state management
> 
> @devprashantt & @AdityaChaudhary18, Any updates from your side?

I'll be starting with the onboarding screen and get it done by tomorrow. Let me know if you need any help in integrating firebase.
